examples/miscellaneous.openapi3.json

```diff
@@ -5818,6 +6220,21 @@
             "X.MovieType": {
                 "title": "MovieType",
                 "type": "object",
+                "properties": {
```

Comment:
I'm confused about this. `properties` and `anyOf`
It doesn't look right to me. But then I found the same in TripPin `Microsoft.OData.SampleService.Models.TripPin.PlanItem`

https://github.com/oasis-tcs/odata-openapi/blob/3a5b2476ab87db662aad69553b4fcf7b64cc42ad/tools/tests/TripPin.openapi3.json#L3660

So it might be correct. But I dont understand the schema then :/

Reply:
This means "these properties, and then the ones from any of the listed other schemas".

Reply:
I see, then it's perhaps not a huge problem. Need to check that it handles duplicate properties correctly

Reply:
Looks good to me
![image](https://user-images.githubusercontent.com/1266245/78180516-2afc8d00-7463-11ea-89e0-98eeb44f7378.png)

Though that empty object seems to be causing a `null` value. But didn't change that - so that's a story for another day :)